#file name: your_parameter_file.yaml
det:
  socket:
    ros__parameters:
      up_rgbd_ins: { 107.282, 107.282,105.0, 60.0}
      up_rgbd_rot: { 0.0,-0.702,0.00 }
      up_rgbd_tra: { 0.365,-0.01,0.03 }

  track:
    ros__parameters:
      front_ins: { 500.2527835897041,716.1117740787954,499.7620978089695,379.9548440735398 }
      front_rot: { 0.0,0.0,0.0}
      front_tra: { 0.253,0.0,0.950 }
      back_ins: { 588.096505105656,660.3054628802164,589.2663187406049,375.4730364531033 }
      back_rot: { 0.0,0.0,0.00 }
      back_tra: { -0.253,0.0,0.950 }
